### PR TITLE
[Auctiva.com] Simplify rule

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/Auctiva.com.xml [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+		betatest.auctiva.com
+		l.e.auctiva.com
+		eduimg.auctiva.com
+		livechat.auctiva.com (broken chain)
 
-		- img.auctiva.com.s3.amazonaws.com
+	No working URL known:
+		counters.auctiva.com
+		scimg.auctiva.com
+		scrollinggallery.auctiva.com
```
Comment (J0WI) [inline src/chrome/content/rules/Auctiva.com.xml]: `http://scrollinggallery.auctiva.com/gallery/redirect/AUCTIVA/1302906/0/6`

File: src/chrome/content/rules/Auctiva.com.xml [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+	Invalid certificate:
+		betatest.auctiva.com
+		l.e.auctiva.com
+		eduimg.auctiva.com
+		livechat.auctiva.com (broken chain)
 
-		- img.auctiva.com.s3.amazonaws.com
+	No working URL known:
+		counters.auctiva.com
+		scimg.auctiva.com
```
Comment (J0WI) [inline src/chrome/content/rules/Auctiva.com.xml]: There is a list of working url on `https://scimg.auctiva.com/` itself.

File: src/chrome/content/rules/Auctiva.com.xml [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
-	CDN buckets:
+	Invalid certificate:
+		betatest.auctiva.com
+		l.e.auctiva.com
+		eduimg.auctiva.com
+		livechat.auctiva.com (broken chain)
 
-		- img.auctiva.com.s3.amazonaws.com
+	No working URL known:
+		counters.auctiva.com
```
Comment (J0WI) [inline src/chrome/content/rules/Auctiva.com.xml]: `http://counters.auctiva.com/aucount/0`

File: src/chrome/content/rules/Auctiva.com.xml (hunk 2 above)
Comment (Bisaloo) [reply src/chrome/content/rules/Auctiva.com.xml]: I still can't find any working URL :(. The content on https://scimg.auctiva.com/ doesn't work.

Comment (J0WI) [reply src/chrome/content/rules/Auctiva.com.xml]: `http://scimg.auctiva.com/imgsc/0/0/0/0/0/8/sc/110024537942.jpg`

**Pull request merged**